content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -0,0 +1,187 @@
+---
+title: 'Environment variables reference'
+metaTitle: 'Prisma environment variables reference'
+metaDescription: 'This page gives an overview of all environment variables avalaible for use.'
+tocDepth: 2
+---
+
+## Overview
+
+This document describes different environment variables and their use cases.
+
+### DATABASE_URL
+
+`DATABASE_URL` is used to connect to your database. The url format can differ depending on the database it is connecting to.
+Prisma currently supports 4 types of database connectors, of which 3 use the `DATABASE_URL` environment variable. (SQLite sources a file path)
+
+- PostgreSQL
+- MySQL
+- SQLite
+- SQLServer
+
+Example using a PostgreSQL connector:
+
+```js file=.env
+DATABASE_URL = 'postgresql://USER:PASSWORD@HOST:PORT/DATABASE'
+```
+
+See [Connection URLs](../database-reference/connection-urls) for a breakdown of each database connector.
+
+### DEBUG
+
+`DEBUG` is used to enable debugging output in the `Prisma Client`.
+
+Example setting `Prisma Client` level debugging output:
+
+```terminal
+# enable only `prisma-client`-level debugging output
+export DEBUG="prisma-client"
+```
+
+See [Debugging](../../concepts/components/prisma-client/debugging) for more information.
+
+### NO_COLOR
+
+`NO_COLOR` strips colors from error messages. It's a truthy value.
```

Comment:
is there another page that we can link to that gives a bit more content and a practical example of this? e.g. here: https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/error-formatting#formatting-via-environment-variables

Reply:
also, just a thought: maybe we can link "truthy" to this page: https://developer.mozilla.org/en-US/docs/Glossary/Truthy

I'm not sure every JS developer (especially newcomers) are familiar with this term so linking this out to its docs might help to make our docs more approachable.

Reply:
> is there another page that we can link to that gives a bit more content and a practical example of this? e.g. here: https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/error-formatting#formatting-via-environment-variables

I link to this env var ref from that article. If i link to it from here it will be like a circle 😅 I guess that doesnt matter, just wanted to point it out.

Good call on the mdn doc link

Reply:
There is also https://no-color.org/